src/server_manager/web_app/test_action.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
We should call `yarn do server_manager/web_app/test` from `src/server_manager/test_action.sh` to run the tests on CI.

Reply:
Done.

@alalamav Should we be running these test actions with the `-eu` flags on Travis so that they exit if a test fails? I added those Bash flags to both test_action scripts.

Reply:
I think that's what we do for the other tests, good catch.